src/Turn.js

```diff
@@ -1,33 +1,39 @@
+import Puzzle from './Puzzle';
```

Comment:
Does this class need all of the other class files? Is a `Round` being instantiated in this file anywhere? Likewise for the other classes. If there is no instantiation of the class, then the class file `import` is probably not needed.